lib/rust/ensogl/component/text/src/component/area.rs

```diff
@@ -29,6 +29,8 @@ use ensogl_core::display;
 use ensogl_core::gui::cursor;
 use ensogl_core::system::web::clipboard;
 use ensogl_core::DEPRECATED_Animation;
+#[cfg_attr(not(target_arch = "wasm32"), allow(unused_imports))]
+use ensogl_text_embedded_fonts as embedded_fonts;
```

Comment:
Why this is done this way?
1. Why this is imported only on wasm32
2. Why there is allos unused_imports?

Reply:
will try 

```
#[cfg(target_arch = "wasm32")]
use ensogl_text_embedded_fonts as embedded_fonts;
```

Reply:
It's imported only on wasm32 because it's used only in code that is compiled only on wasm32. The conditional compilation is needed for tests, as not all functionalities of the browser APIs are wrapped for tests.

As to unused_imports, I changed the attributes to the more common and simpler style of `#[cfg(target = "wasm32)]` - it seems to work fine.